staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -335,13 +336,22 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 		return r.base.RoundTrip(req)
 	}
 
-	creds, err := r.a.getCreds()
+	creds, _, err := r.a.getCreds()
 	if err != nil {
 		return nil, fmt.Errorf("getting credentials: %v", err)
 	}
 	if creds.token != "" {
+		req = utilnet.CloneRequest(req)
```

Comment:
> did you just notice this or did a test catch it?

I just noticed it as I was changing the code.  No test failed.

> would we want to hoist this bugfix back independent of the rest of this PR?

It would not hurt to backport though I am unsure if it matters in practice.